mackup/applications/rstudio.cfg

```diff
@@ -4,3 +4,4 @@ name = RStudio
 [configuration_files]
 .rstudio-desktop
 Library/Preferences/org.rstudio.RStudio.plist
+.config/rstudio
```

Comment:
To support Freedesktop standard, It should be
```
[xdg_configuration_files]
rstudio
```

e.g. https://github.com/lra/mackup/blob/755095e590deab8f4a2875e36800a021094f683d/mackup/applications/git.cfg

Reply:
Thanks - was not aware of this. Updated in 1ce5dde